packages/interim/src/renovation/component_wrapper/common/component.ts

```diff
@@ -97,16 +97,14 @@ export default class ComponentWrapper extends DOMComponent<ComponentWrapperProps
   }
 
   validateKeyDownHandler(): void {
-    /* const supportedKeyNames = this.getSupportedKeyNames();
+    const supportedKeyNames = this.getSupportedKeyNames();
```

Comment:
@MikeVitik, is it OK that I uncommented these lines?

Reply:
Yes. It's OK